Main/styles/common/_tweet-dialog.scss

```diff
@@ -4,61 +4,71 @@
     left: 0;
     width: 100%;
     height: 100%;
-
     display: flex;
     align-items: center;
     justify-content: center;
-
     z-index: 400;
     background-color: #00000080;
+}
 
-    .tweet-dialog {
-        background-color: white;
-        border-radius: 4px;
-        box-shadow: 0px 2px 10px 0px #0000001A;
-        padding: 8px 16px;
-
-        display: flex;
-        flex-direction: column;
-        gap: 10px;
-
-        h1 {
-            font-family: Pretendard;
-            font-size: 18px;
-            font-weight: 700;
-            line-height: 21px;
-            letter-spacing: -0.01em;
-            text-align: left;
-        }
-
-        textarea {
-            font-family: Pretendard;
-            font-size: 14px;
-            font-weight: 400;
-            line-height: 17px;
-            letter-spacing: -0.01em;
-            text-align: left;
+.tweet-dialog {
+    background-color: white;
+    border-radius: 8px;
+    box-shadow: 0px 2px 10px 0px #0000001A;
+    padding: 16px;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    height: 200px;
+}
 
-            border-radius: 4px;
-            border: 1px solid #DFDFDF;
+.tweet-dialog h1 {
+    font-family: Pretendard;
+    font-size: 24px;
+    font-weight: 300;
+    line-height: 28px;
+    letter-spacing: -0.01em;
+    text-align: left;
+    color: #53afe8;
+}
 
-            height: 10em;
-            max-height: 80vh;
-            width: 25em;
-            max-width: 100%;
-        }
+.tweet-dialog textarea {
+    font-family: Pretendard;
+    font-size: 18px;
+    font-weight: 300;
+    line-height: 22px;
+    letter-spacing: -0.01em;
+    text-align: left;
+    border-radius: 4px;
+    border: 1px solid #53afe8;
+    
+    background-color: white;
+    height: calc(100% - 50px);
+    max-height: 80vh;
+    width: 30em;
+    max-width: 100%;
+    padding: 8px;
+}
 
-        button {
-            border-radius: 4px;
-            padding: 6px 12px;
-            border: none;
+.tweet-dialog button {
+    border-radius: 10px; 
+    padding: 8px 24px;
+    font-family: Pretendard;
+    font-size: 20px;
+    font-weight: 300;
+    border: none; 
+    line-height: 24px;
+    letter-spacing: -0.01em;
+    text-align: center;
+    background-color: #1DA1F2;
+    color: white;
+    transition: background-color 0.3s, transform 0.3s;
+}
 
-            font-family: Pretendard;
-            font-size: 16px;
-            font-weight: 400;
-            line-height: 19px;
-            letter-spacing: -0.01em;
-            text-align: left;
-        }
-    }
-}
\ No newline at end of file
+.tweet-dialog button:hover {
```

Comment:
Please use nesting feature of scss

For example, write code in A style, not in B style.

```scss
// A
.example {
    lorem: ipsum;

    &:hover {
        example: something;
    }
}
```

```scss
// B
.example {
    lorem: ipsum;
}

.example:hover {
    example: something;
}
```